src/events.js

```diff
@@ -268,148 +274,58 @@ export const addEventAddSpecimensToListModalButton = (bagId, tableIndex, isOrpha
-                    continue;
-                } else {
-                    tubesToAdd.mouthWash.push(currTubeNum);
-                }
-            } else {
-                if (tubesInBox.shipped.bloodUrine.includes(currTubeNum) || tubesInBox.shipped.orphan.includes(currTubeNum) || tubesInBox.notShipped.bloodUrine.includes(currTubeNum) || tubesInBox.notShipped.orphan.includes(currTubeNum)) {
-                    continue;
+            const boxToUpdate = prepareBoxToUpdate(currBoxId, boxList, boxIdAndBagsObj, locations, addedTubes);
+            showAnimation();
+            try {
+                const boxUpdateResponse = await updateBox(boxToUpdate);
+                hideAnimation();
+                if (boxUpdateResponse.code === 200) {
+                    updateShippingStateAddBagToBox(currBoxId, bagId, boxToUpdate, boxUpdateResponse.data);
+                    await startShipping(appState.getState().userName, true, currBoxId);
                 } else {
-                    tubesToAdd.bloodUrine.push(currTubeNum);
+                    console.error('Failed to update box.', boxUpdateResponse);
+                    showNotifications({ title: 'Error Adding Specimen(s)', body: `There was an error adding ${bagId.split(' ')[0]}. Please try again.` });
                 }
+            } catch (error) {
+                hideAnimation();
+                console.error('Failed to update box.', error);
+                showNotifications({ title: 'Error Adding Specimen(s)', body: `There was an error adding ${bagId.split(' ')[0]}. Please try again.` });
             }
         }
+    }, { once: true })
+}
 
-        if ((tubesInBox.shipped.bloodUrine.length > 0 || tubesInBox.notShipped.bloodUrine.length > 0) && tubesToAdd.bloodUrine.length > 0) {
-            tubesToAdd.orphan=tubesToAdd.bloodUrine;
-            tubesToAdd.bloodUrine=[];
-        }
-
-        for (const tubeNum of tubesToAdd.orphan) {
-            if (!availableCollectionsObj['unlabelled']) {
-                availableCollectionsObj['unlabelled'] = [];
-            }
-            availableCollectionsObj['unlabelled'].push(currCollection[conceptIds.collection.id] + ' ' + tubeNum);
-        }
-
-        if (tubesInBox.shipped.bloodUrine.length === 0 && tubesInBox.notShipped.bloodUrine.length ===0 && tubesToAdd.bloodUrine.length> 0) {
-            availableCollectionsObj[currCollection[conceptIds.collection.id] + ' 0008'] = tubesToAdd.bloodUrine;
-        }
-
-        if (tubesInBox.shipped.mouthWash.length === 0 && tubesInBox.notShipped.mouthWash.length ===0 && tubesToAdd.mouthWash.length > 0) {
-            availableCollectionsObj[currCollection[conceptIds.collection.id] + ' 0009'] = tubesToAdd.mouthWash;
-        }
+const checkBagCount = (bagCount, bagId, currBoxId) => {
+    const maxBoxSize = bagConceptIdList.length;
+    const timeToAlert = maxBoxSize - 3;
```

Comment:
I thought about this, too. I hesitated to create the file since it's the only constant I've used...happy to create the file and reference if it's a better way to go.